src/test/java/uk/ac/cam/cl/dtg/isaac/api/ITConstants.java

```diff
@@ -36,15 +38,32 @@ public final class ITConstants {
   public static final long TEST_TEACHER_ID = 5L;
 
   public static final String DAVE_TEACHER_EMAIL = "dave-teacher@test.com";
+  public static final String DAVE_TEACHER_PASSWORD = "test1234";
 
   public static final String TEST_STUDENT_EMAIL = "test-student@test.com";
   public static final String TEST_STUDENT_PASSWORD = "test1234";
   public static final long TEST_STUDENT_ID = 6L;
 
+  public static final String TEST_STUDENT_ALICE_EMAIL = "alice-student@test.com";
```

Comment:
What's the reason to have these new test users' names as part of the variable? What makes Alice, Bob, Erika and Charlie different to the test student with ID 6? Are they just additional students? If so, you may want to name them student 2, student 3, etc. Just because you already have a test student, so it looks strange to have one unnamed student and then all these named students. If, on the other hand, there is something special about these new users, then that may be a sign that the name should reflect that.

Reply:
Those are the pre-existing names used in the test database, I just followed the naming pattern established by the other entries.
I don't know why they used the generic alphabet names rather than numbers but I can have a look into changing it if you'd prefer?

Reply:
My only complain about this is that I cannot tell just by looking at this why are they needed. Why cannot we just use the existing`TEST_STUDENT_EMAIL`? If there is a special reason why we need these new ones, then these constant names should reflect that reason, so that it's evident why they're here. For instance, `TEST_STUDENT_EMAIL` is pretty clear: it's the email of a test student. But `TEST_STUDENT_ALICE_EMAIL` is not a great name because it does not tell me why a student named Alice is needed, if that makes sense.

Reply:
Fair enough, I'll take another look at the constant names and see if I can distill it down to something more explanatory...

Reply:
Okay, I've changed the constants a bit to try an improve the clarity. Also added some supplementary explanation in comments, since it would be either unclear or unwieldy to try and put the whole thing in the constant.
Let me know if you have any suggestions to improve it further.

Reply:
Thank you very much for changing this, I think it is easier to understand now.
I think that the names would make more sense if they read like the following: for instance, instead of `QUIZ_FACADE_TEST_STUDENT_3_EMAIL_WITH_FREE_ATTEMPT` it should be `QUIZ_FACADE_TEST_STUDENT_3_WITH_FREE_ATTEMPT_EMAIL`. What do you think?

Reply:
I wasn't sure which way round to put those bits myself, so if you think that would be better, I'm happy to swap them over. 👍